pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 dependencies = [
-    "llama-stack @ git+https://github.com/cdoern/llama-stack.git@feat-658",
+    "llama-stack @ git+https://github.com/astefanutti/llama-stack.git@feat-658-1238",
```

Comment:
Why are we making this change? Why don't we use the https://github.com/meta-llama/llama-stack/pull/1672

Reply:
nvm we need the job/async code too.
Just curious why not use Charlie branche anymore?

Reply:
Right https://github.com/booxter/llama-stack/tree/scheduler-for-torchtune is needed.

@cdoern branch isn't rebased on top of your latest changes.